sdk/cognitiveservices/Language.LUIS.Authoring/src/Generated/IModel.cs

```diff
@@ -40,7 +40,7 @@ public partial interface IModel
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        /// <exception cref="Microsoft.Rest.HttpOperationException">
+        /// <exception cref="ErrorResponseException">
```

Comment:
Is this a breaking change or just a fix to the documentation?

Reply:
I'll revert this change for now and just support the roles in this PR